Consumer/ConsumerManager.php

```diff
@@ -0,0 +1,210 @@
+<?php
+namespace M6Web\Bundle\KafkaBundle\Consumer;
+
+use M6Web\Bundle\KafkaBundle\AbstractKafka;
+
+/**
+ * Class Consumer
+ * @package M6Web\Bundle\KafkaBundle\Consumer
+ *
+ * A class to consume messages with set or unset topics
+ */
+class ConsumerManager extends AbstractKafka
+{
+    const ORIGIN = 'consumer';
+
+    /**
+     * @var array
+     */
+    protected $topicsFromServer;
+
+    /**
+     * @var \RdKafaka\Consumer
+     */
+    protected $consumer;
+
+    /**
+     * @var array
+     */
+    protected $topics = [];
+
+    /**
+     * @var array
+     */
+    protected $topicsConsumptionState = [];
+
+    /**
+     * @var Topic
+     */
+    protected $newTopicToConsume;
+
+    /**
+     * @var Queue
+     */
+    protected $queue;
+
+    /**
+     * @param \RdKfaka\Consumer $consumer
+     *
+     * @return ConsumerManager
+     */
+    public function setConsumer(\RdKafka\Consumer $consumer)
+    {
+        $this->consumer = $consumer;
+
+        return $this;
+    }
+
+    /**
+     * @param integer $logLevel
+     *
+     * @return ConsumerManager
+     */
+    public function setLogLevel(int $logLevel)
+    {
+        $this->consumer->setLogLevel($logLevel);
+
+        return $this;
+    }
+
+    /**
+     * @param string $brokers
+     *
+     * @return ConsumerManager
+     */
+    public function addBrokers(string $brokers)
+    {
+        $this->consumer->addBrokers($brokers);
+
+        return $this;
+    }
+
+    /**
+     * @param string $name
+     * @param array  $confs
+     *
+     * @return void
+     */
+    public function addTopic(string $name, array $confs = [])
+    {
+        $topicConf = new \RdKafka\TopicConf();
+
+        array_walk($confs, function ($conf, $confIndex) use ($topicConf) {
+            $topicConf->set($confIndex, $conf);
+        });
```

Comment:
Same remark than previously, I'm not sure that in this case `array_walk` with a short anonymous function is more efficient/clear than a *simple* `foreach`.